package.json

```diff
@@ -33,7 +33,7 @@
     "babel-polyfill": "^6.3.14",
     "babel-preset-es2015": "^6.3.13",
     "babel-register": "^6.4.3",
-    "builder": "~2.5.0",
+    "builder": "~2.6.0",
```

Comment:
Not true, @greenkeeperio-bot; [Tilde-ranges](https://docs.npmjs.com/misc/semver#tilde-ranges-1-2-3-1-2-1) allow for minor updates. Still, we should switch this to a caret `^` range.